swiftgen-cli/storyboards.swift

```diff
@@ -15,19 +15,21 @@ let storyboardsCommand = command(
     description: "The name of the enum to generate for Scenes"),
   Option<String>("segueEnumName", "StoryboardSegue", flag: "g",
     description: "The name of the enum to generate for Segues"),
-  Options<String>("import", [], count: 0,
+  VariadicOption<String>("import", [],
```

Comment:
Yeah, you can imagine how I felt when I tried the release 😱 

I'd completely misinterpreted the purpose of `Options<>` which needs a count for the number of arguments it should expect. This fixes that issue 😅